### PR TITLE
Prepare docs for Doctrine website

### PR DESCRIPTION
**Pull request by driehle**
Similar to and in line with:

- https://github.com/doctrine/DoctrineModule/pull/733
- https://github.com/doctrine/DoctrineORMModule/pull/672
- https://github.com/doctrine/doctrine-laminas-hydrator/pull/34

### CODE REVIEW

File: docs/en/development.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+::
+
+   docker-compose run --rm php composer test-coverage
+
+Run phpcs as
+
+::
+
+   docker-compose run --rm php composer cs-check src test config
```
Comment (SenseException) [inline docs/en/development.rst]: Maybe the code highlighting for bash would look nice here.

**Review by SenseException — CHANGES_REQUESTED** [review]

File: docs/en/usage.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
+Usage
+=====
+
+Command Line
+------------
+
+Access the Doctrine command line as following
```
Comment (TomHAnderson) [inline docs/en/usage.rst]: `...as follows`

File: docs/en/usage.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+   php public/index.php
+
+or
+
+.. code:: sh
+
+   ./vendor/bin/doctrine-module
+
+Service Locator
```
Comment (TomHAnderson) [inline docs/en/usage.rst]: `Service Manager`

Service locator is no longer in the vernacular for this context.

File: docs/en/user-guide.rst [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+==========
+
+This guide helps you to create a Laminas application with Doctrine Mongo
+ODM integration. If you’re new to Laminas, please read the `Laminas
+documentation <https://docs.laminas.dev/>`__ before you continue.
+
+Install Composer
+----------------
+
+install composer via ``curl -s http://getcomposer.org/installer | php``
```
Comment (TomHAnderson) [inline docs/en/user-guide.rst]: Capitalize `Install`

File: docs/en/user-guide.rst [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+   $ php -S 0.0.0.0:8080 -t public/ public/index.php
+
+For detailed instructions on installing the Laminas MVC Skeleton
+Application follow `this
+link <https://github.com/laminas/laminas-mvc-skeleton>`__.
+
+Install Doctrine Mongo ODM Module
+---------------------------------
+
+Install DoctrineODMModule :
```
Comment (TomHAnderson) [inline docs/en/user-guide.rst]: `Install DoctrineMongoODMModule :`

File: docs/en/user-guide.rst [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+.. code:: bash
+
+   $ composer require doctrine/doctrine-mongo-odm-module
+
+Open ``doctrine-odm-tutorial/configs/application.config.php`` in your
```
Comment (TomHAnderson) [inline docs/en/user-guide.rst]: doctrine-odm-tutorial?  That sounds like a very old reference and is certainly out of scope for Laminas.  Just reference `config/application.config.php`

**Review by TomHAnderson — CHANGES_REQUESTED** [review]

**Review by TomHAnderson — APPROVED** [review]

**Pull request merged**